geode-core/src/test/java/org/apache/geode/internal/cache/PartitionedRegionClearTest.java

```diff
@@ -56,15 +57,19 @@
 
 public class PartitionedRegionClearTest {
 
+  private GemFireCacheImpl cache;
+  private HashSet<AsyncEventQueue> allAEQs = new HashSet<>();
+  private PartitionedRegionClear partitionedRegionClear;
   private DistributionManager distributionManager;
   private PartitionedRegion partitionedRegion;
   private RegionAdvisor regionAdvisor;
   private InternalDistributedMember internalDistributedMember;
 
-  private PartitionedRegionClear partitionedRegionClear;
-
   @Before
   public void setUp() {
+
+    cache = mock(GemFireCacheImpl.class);
+    partitionedRegion = mock(PartitionedRegion.class);
```

Comment:
`partitionedRegion` is already defined on line 75, so this is redundant.

Reply:
caused by merge. kirk added today.